core/client/fs/src/test/java/alluxio/client/file/FileInStreamTest.java

```diff
@@ -336,9 +336,18 @@ public void testSeekWithNoLocalWorker() throws IOException {
       Assert.assertEquals(readAmount, mInStreams.get(0).getBytesRead());
       Assert.assertEquals(0, mInStreams.get(1).getBytesRead());
     }
+  }
 
-    // nothing is written to cache stream
-    Assert.assertEquals(0, mCacheStreams.get(0).getWrittenData().length);
+  @Test
+  public void seekAndClose() throws IOException {
```

Comment:
Should we test the negative case?

Reply:
Can you be more specific?

Reply:
the case this PR is trying to address, unnecessarily caching when partial caching is on

Reply:
Yes, this test `seekAndClose` will fail without this fix. It fails on the source of `Local`.